BuildingBlocks/src/BuildingBlocks.Infrastructure/EventBus/RabbitMQ/EventBusRabbitMq.cs

```diff
@@ -244,8 +238,7 @@ public async Task Publish(DomainEvent @event)
         var eventName = @event.GetEventName();
 
         _logger.LogInformation("Creating RabbitMQ channel to publish a '{EventName}'.", eventName);
-
-        var message = JsonConvert.SerializeObject(@event, JSON_SERIALIZER_SETTINGS);
+        var message = JsonSerializer.Serialize(@event, @event.GetType());
```

Comment:
The autistic Me forces me to write this comment:

Can you re-add the blank line between this one and the LogInformation please? 😉